discord/guild.py

```diff
@@ -2907,8 +2907,9 @@ async def ban(
         self,
         user: Snowflake,
         *,
+        delete_message_days: Optional[int] = 0,
+        delete_message_seconds: Optional[int] = 0,
```

Comment:
Should we default these to None since they're optional? The API will default them to 0 anyways, and sending less data would reduce the payload.